src/main/java/com/chatcode/config/dummy/DummyDevInit.java

```diff
@@ -103,9 +107,54 @@ CommandLineRunner dummyInterestTags(InterestTagReadRepository interestTagReadRep
 
 
     @Bean
-    CommandLineRunner dummyArticles() {
+    CommandLineRunner dummyArticles(ArticleWriteRepository articleWriteRepository,
```

Comment:
dummyArticles에 등록하는 유저를 dummyUsers와 연결하는 건 어떻게 생각하시나요?!!
```suggestion
    @DependsOn(value = {"dummyUsers"})
    CommandLineRunner dummyArticles(ArticleWriteRepository articleWriteRepository,
```
찾아보니까, @DependsOn(value = {"dummyUsers"}) 을 사용하면 등록된 테스트 유저를 가져올 수 있을 것 같아서요!

Reply:
좋은 것 같습니다~! 준현님 덕분에 몰랐던 어노테이션 알고가네요